src/index.ts

```diff
@@ -157,6 +157,21 @@ app.get('/profile/:user_id?', async (req: Request, res: Response) => {
     res.status(200).json(user);
 });
 
+// Post route to update profile information of user_id from users table.
+app.post('/profile/edit/:user_id', async (req: Request, res: Response) => {
```

Comment:
Change route name from `/profile/edit/:user_id` to `/profile/edit`. The `user_id` is not needed because it is stored in the JSON Web Token. Since you can only edit your own profile (for security), we do not need `user_id` in the route name.